django_redis/client/default.py

```diff
@@ -205,7 +205,7 @@ def incr_version(
         value = self.get(old_key, version=version, client=client)
 
         try:
-            ttl = client.ttl(old_key)
+            ttl = self.ttl(old_key, version=version, client=client)
```

Comment:
I assume make_key isn't expensive? (it's going to be called twice now). The final comment I have is TTL could still be returned as 0. That's ok for this function, right? Not sure if that needs to be handled at all (I'm assuming it is, but just calling it out)

Reply:
it needs to be called because the old key has to be deleted

Reply:
line 221

Reply:
Ack, I understand what this does better, thanks